tests/python/contrib/test_ethosn/test_fullyconnected.py

```diff
@@ -30,7 +32,11 @@ def _get_model(
 ):
     """Return a model an any parameters it may have"""
     a = relay.var("a", shape=shape, dtype=dtype)
-    weights_array = tvm.nd.array(np.ones(weight_shape, dtype))
+    weights_array = tvm.nd.array(
+        np.random.randint(
+            np.iinfo(dtype).min, high=np.iinfo(dtype).max, size=weight_shape, dtype=dtype
```

Comment:
Would it be better to have a separate function providing range based on dtype? We do have something similar is microNPU?

Reply:
Agreed we should pull this out into common functionality, I'll do this in a follow up patch